versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractITVersionStore.java

```diff
@@ -150,7 +154,8 @@ public void createAndDeleteBranch() throws Exception {
     assertEquals(commitHash, otherCreateHash);
 
     List<WithHash<NamedRef>> namedRefs;
-    try (Stream<WithHash<NamedRef>> str = store().getNamedRefs()) {
+    try (Stream<WithHash<NamedRef>> str =
+        store().getNamedRefs().filter(r -> !r.getValue().getName().equals("main"))) {
```

Comment:
nit: Could we abstract this filter into an overridable method, perhaps, to make it clearer that we exclude `main` only in adapter-based  tests?